website/docs/tutorials/02-installation.md

```diff
@@ -4,45 +4,50 @@ sidebar_label: Installation
 
 # Installation
 
+Protostar is a valuable tool for developing StarkNet smart contracts, and it is easy to install on Linux and Mac systems. In this section, we will walk through the process of installing and upgrading Protostar, as well as the requirements for using the tool.
+
 ## Requirements
-Protostar requires a [git](https://git-scm.com/) executable to be specified in the `$PATH`.
+To use Protostar's dependency management commands, you must have git installed and added to your `$PATH` environment variable.
+
+The `$PATH` is an environment variable that specifies a list of directories in which the system looks for executables.
```

Comment:
```suggestion
```
Not relevant

Reply:
It's relevant, but it's quite likely somebody interested in L2 smart contracts already knows what PATH is. Removing.